x-pack/test/api_integration/apis/upgrade_assistant/index.ts

```diff
@@ -9,14 +9,13 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('Upgrade Assistant', function () {
```

Comment:
While reviewing this I noticed that `node_disk_space` test is not loaded in this file. Do you know why that happened?  I think we could either load it and add the `this.onlyEsVersion('<=7');` or get rid of it if its not needed. WDYT?

Reply:
Great catch! I added it back to `index.ts`. Not sure how/when that happened.